onload/dev/kustomization.yaml

```diff
@@ -6,22 +6,3 @@ namePrefix: dev-
 resources:
 - ../base
 
-patches:
```

Comment:
What adds `--debug` instead?

Reply:
This is just something I forgot to restore from previous testing, I'll add in back in.